### PR TITLE
Rating & reviews

### PR DESCRIPTION
**Pull request by TammamTautou**
I believe all functionalities are working

### CODE REVIEW

File: client/src/components/Ratings/sub-comps/WriteNewReview.jsx
Comment (lundas) [inline client/src/components/Ratings/sub-comps/WriteNewReview.jsx:1]: keep an eye out for these jsx-ally eslint-disables. eventually need to include those features

**Review by lundas — COMMENTED** [review]
Since this one is such a large PR, gonna also give @Benjamin-Cardon a chance to review before approving

Made some minor comments on individual files. Biggest thing is keep an eye out of those disabled linter comments for the a11y stuff as that's functionality we'll eventually need in the final product

File: client/dist/style.css
[the remark this answers is not in the document]
Comment (TammamTautou) [reply client/dist/style.css:1]: I agree those are just for testing how it would look

File: client/src/components/Ratings/sub-comps/ReviewTile.jsx
Comment (Benjamin-Cardon) [inline client/src/components/Ratings/sub-comps/ReviewTile.jsx:1]: Also for conciseness and unity, we probably should all use the same package for dates. This is a feature which I haven't implemented yet, so I'll be following your example and using what you've used.

**Review by lundas — APPROVED** [review]

**Review by Benjamin-Cardon — APPROVED** [review]
In general, It's a good submission.
Andrew already brought up the two concerns which I think are relevant. The first is unity of CSS, the other is having disabled a lot of ESlint errors. I think we'll be able to solve both of them very organically with the flow of our work, it's just a reminder to not get too attached to any particular part of code.

**Pull request merged**